.github/workflows/ci.yml

```diff
@@ -20,9 +20,20 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - name: Install native dependencies
         run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev
-      - run: cargo check --workspace 
+      - run: cargo check --workspace
+      - run: cargo check --features=hot_reload,buffer_labels
       # --exclude with_bevy # for when bevy has an outdated wgpu version
       # -Dwarnings # for when we have fixed unused code warnings
+
+  clippy:
+    runs-on: ubuntu-latest
+    name: Enforce clippy lints
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dtolnay/rust-toolchain@stable
+      - name: Install native dependencies
+        run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev
+      - run: cargo clippy --all-targets --workspace -- -D warnings
```

Comment:
My understanding is that `cargo clippy` is strictly more strict than `cargo check`, so we don't need a job for both.
I think we can merge both these jobs (maybe renaming the job?)

If we do rename, we'd need to change the branch protection rules to allow this to land.

Reply:
Right. But there is still some discussion to be had about the exact build matrix. Arguably we should check build success in a variety of configurations, but perhaps be strict about warnings and lints only in a smaller number (this may allow us to reduce the `[#allow]` instances that are ascribed to cfg changes.